vllm/v1/core/scheduler_output.py

```diff
@@ -32,18 +37,18 @@ def from_request(
         block_ids: List[int],
         num_computed_tokens: int,
     ) -> "NewRequestData":
-        return cls(
-            req_id=request.request_id,
-            prompt_token_ids=request.prompt_token_ids,
-            prompt=request.prompt,
-            mm_inputs=request.mm_inputs,
-            mm_hashes=request.mm_hashes,
-            mm_positions=request.mm_positions,
-            sampling_params=request.sampling_params,
-            block_ids=block_ids,
-            num_computed_tokens=num_computed_tokens,
-            lora_request=request.lora_request,
-        )
+        return cls(req_id=request.request_id,
+                   prompt_token_ids=request.prompt_token_ids,
+                   prompt=request.prompt,
+                   mm_inputs=request.mm_inputs,
+                   mm_hashes=request.mm_hashes,
+                   mm_positions=request.mm_positions,
+                   sampling_params=request.sampling_params,
+                   block_ids=block_ids,
+                   num_computed_tokens=num_computed_tokens,
+                   lora_request=request.lora_request,
+                   grammar=request.grammar,
+                   grammar_bitmask=request.grammar_bitmask)
```

Comment:
nit: you can minimize the code change here by adding `,` at the end.
```suggestion
                   grammar_bitmask=request.grammar_bitmask,
               )
```